internal/model/model.go

```diff
@@ -30,8 +30,9 @@ type Bookmark struct {
 
 // Account is person that allowed to access web interface.
 type Account struct {
-	ID       int    `db:"id"       json:"id"`
-	Username string `db:"username" json:"username"`
-	Password string `db:"password" json:"password,omitempty"`
-	Owner    bool   `db:"owner"    json:"owner"`
+	ID       int    `db:"id"                   json:"id"`
+	Username string `db:"username"             json:"username"`
+	Password string `db:"password"             json:"password,omitempty"`
+	Owner    bool   `db:"owner"                json:"owner"`
+	Config   string `db:"config"               json:"config"`
```

Comment:
Have you checked https://github.com/go-shiori/shiori/pull/639#discussion_r1265419217 ?

Reply:
In your latest approach, from what I can see you can remove the empty slice check (since `for rows.Next()` wont do anything if the result set is empty anyway).

Also, if you use `Queryx` instead of `Query`, you can use `rows.StructScan` later in the for loop, which will help using a struct for the configuration as well.

Does this help?